README.md

```diff
@@ -65,7 +65,7 @@ This repo does not include any assets or assembly code necessary for compiling t
 | BOSS/BO2/BO2.BIN | ![code coverage BO2.BIN](https://img.shields.io/endpoint?label=BO2%20code&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Fcode%2F%3Fmode%3Dshield%26measure%3Dbobo2) | ![decompiled functions](https://img.shields.io/endpoint?label=BO2%20funcs&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Ffunctions%2F%3Fmode%3Dshield%26measure%3Dbobo2) | Minotaur & Werewolf
 | BOSS/BO3/BO3.BIN | ![code coverage BO3.BIN](https://img.shields.io/endpoint?label=BO3%20code&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Fcode%2F%3Fmode%3Dshield%26measure%3Dbobo3) | ![decompiled functions](https://img.shields.io/endpoint?label=BO3%20funcs&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Ffunctions%2F%3Fmode%3Dshield%26measure%3Dbobo3) | Scylla
 | BOSS/BO4/BO4.BIN | ![code coverage BO4.BIN](https://img.shields.io/endpoint?label=BO4%20code&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Fcode%2F%3Fmode%3Dshield%26measure%3Dbobo4) | ![decompiled functions](https://img.shields.io/endpoint?label=B04%20funcs&url=https%3A%2F%2Fprogress.deco.mp%2Fdata%2Fsotn%2Fus%2Ffunctions%2F%3Fmode%3Dshield%26measure%3Dbobo4) | Doppleganger 10
```

Comment:
Since you're already in the file, label=B04 instead of BO4.
I can't add a comment to RBO4, but it's label is RBO3 instead of RBO4.